docs/source/filesystems.mdx

```diff
@@ -191,7 +191,7 @@ Remember to define your credentials in your [FileSystem instance](#set-up-your-c
 List files from a cloud storage with your FileSystem instance `fs`, using `fs.ls`:
 
 ```py
->>> fs.ls("my-private-datasets/imdb/train")
+>>> fs.ls("my-private-datasets/imdb/train", detail=False)
```

Comment:
This is indeed an example about `fsspec` we provide in our docs, but we really do not use `fs.ls` in the library. I agree that the example is informative/useful for the end user though...